### PR TITLE
Update softethervpn.spec

### PR DESCRIPTION
**Pull request by kot1grun**
Update upstream to 4.22.9634-beta.
spec modified for building for several releases (i.e. EL5, EL6, Fedora)

### CODE REVIEW

**Comment by moatazelmasry2** [comment]
Could you please rebase your changesesontop of the current master branch. leave the versio (major, minor etc..) as they are in the master branch? I'd then be happy to merge your changes

File: centos/SPECS/softethervpn.spec [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Name:           softethervpn
 Version:        %{majorversion}.%{minorversion}.%{buildversion}
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (moatazelmasry2) [inline centos/SPECS/softethervpn.spec]: Please keep the release at "1".

**Comment by moatazelmasry2** [comment]
PR #518 has been merged which introduces CMake. Please use this mechanism instead of mak files in your centos/ directory

**Comment by chipitsine** [comment]
@kot1grun , are you still here ?
sorry for a long delay, we are finally going to do something with this PR

are you with us ?

**Comment by kot1grun** [comment]
Hello there. Yes I'm still here, terribly sorry for delay, have ongoing
project which takes all my time. I'll try to make rebase today.

2018-07-21 8:56 GMT+03:00 Ilya Shipitsin <notifications@github.com>:

**Comment by chipitsine** [comment]
besides rebasing - couple of things

1) patch acceptance policy: http://www.softether.org/5-download/src/9.patch (please specify 1 or 2)
2) build system moved to cmake (anyway, spec is broken now, so I'm going to merge your PR even if is just rebased to master)

**Comment by kot1grun** [comment]
1 - sure thing
2 - I can make modifications to support CMake, need some tests though

2018-07-23 13:07 GMT+03:00 Ilya Shipitsin <notifications@github.com>:

**Comment by chipitsine** [comment]
it would be definetely nice if you will adopt cmake as well ))

"1 - sure thing" is not clear enough

**Comment by chipitsine** [comment]
@kot1grun , are you here ?

**Comment by chipitsine** [comment]
rpm build system has changed drastically
CPack was introduced in https://github.com/SoftEtherVPN/SoftEtherVPN/pull/648

sorry, I have to close this PR
thank you for contribution!

**Pull request closed without merging**